.openshift-ci/build/generate-db-dump.sh

```diff
@@ -12,10 +12,10 @@ gate_job generate-db-dump
 generate_db_dump() {
     info "Generating DB dump"
 
-    groupadd -g 71 pg
-    adduser pg -u 71 -g 71 -d /var/lib/postgresql -s /bin/sh
+    groupadd -g 1001 pg
+    adduser pg -u 1001 -g 1001 -d /var/lib/postgresql -s /bin/sh
 
-    runuser -l pg -c "$ROOT/scripts/ci/postgres.sh start_postgres"
+    runuser -l pg -c "PATH=$PATH:/usr/pgsql-$PG_MAJOR/bin/ $ROOT/scripts/ci/postgres.sh start_postgres"
```

Comment:
added a comment

Reply:
Please, use `PATH=$PATH` instead of `PATH=$PATH:/usr/pgsql-$PG_MAJOR/bin/`. First, because the base image already set up the PATH correctly and it's easier to leave that knowledge in one place. Second, it's redundant.